### PR TITLE
Don't display warning if module disabled

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Don't display warning if module disabled
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/23941
| How to test?  | See issue
| Sponsor company| @Touxten

### CODE REVIEW

File: ps_wirepayment.php [hunk 1]
```diff
@@ -69,10 +69,10 @@ public function __construct()
         $this->displayName = $this->trans('Wire payment', array(), 'Modules.Wirepayment.Admin');
         $this->description = $this->trans('Accept wire payments by displaying your account details during the checkout.', array(), 'Modules.Wirepayment.Admin');
         $this->confirmUninstall = $this->trans('Are you sure about removing these details?', array(), 'Modules.Wirepayment.Admin');
-        if (!isset($this->owner) || !isset($this->details) || !isset($this->address)) {
+        if ((!isset($this->owner) || !isset($this->details) || !isset($this->address)) && 1 == $this->active) {
```
Comment (PierreRambaud) [inline ps_wirepayment.php:72]:
```suggestion
        if ((!isset($this->owner) || !isset($this->details) || !isset($this->address)) && $this->active) {
```

File: ps_wirepayment.php [hunk 2]
```diff
@@ -69,10 +69,10 @@ public function __construct()
...
             $this->warning = $this->trans('Account owner and account details must be configured before using this module.', array(), 'Modules.Wirepayment.Admin');
         }
-        if (!count(Currency::checkPaymentCurrencies($this->id))) {
+        if (!count(Currency::checkPaymentCurrencies($this->id)) && 1 == $this->active) {
```
Comment (PierreRambaud) [inline ps_wirepayment.php:75]:
```suggestion
        if (!count(Currency::checkPaymentCurrencies($this->id)) && $this->active) {
```

**Comment by okom3pom** [comment]
Hi @PierreRambaud i made the same change for https://github.com/PrestaShop/ps_checkpayment/pull/48 thank you

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by aDamelincourt** [comment]
Thanks @okom3pom 
✔️  QA approved
No more alert when module is disabled (and no configured).

**Comment by Progi1984** [comment]
Thanks @okom3pom & @aDamelincourt

**Pull request merged**